avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -757,70 +753,82 @@ def updateFieldsC(cfg, particles, dem, fields):
 
   return particles, fields
 
+
 @cython.boundscheck(False)
 @cython.wraparound(False)
 @cython.cdivision(True)
 def getNeighboursC(particles, dem):
-    """ Locate particles in cell for SPH computation (for loop implementation)
+    """ Locate particles on DEM and SPH grid
 
-    Ĺocate each particle in a grid cell and build the indPartInCell and
-    partInCell arrays. See issue #200 and documentation for details
+    Ĺocate each particle in a grid cell (both DEM and SPH grid) and build the
+    indPartInCell and partInCell arrays for SPH neighbourSearch and
+    InCell, IndX and IndY arrays location on the DEM grid.
+    See issue #200 and documentation for details
 
     Parameters
     ----------
     particles : dict
     dem : dict
+      dem dict with SPH header (information about SPH grid)
 
     Returns
     -------
     particles : dict
-      updated particles dictionary with indPartInCell and partInCell arrays
+      updated particles dictionary with
+      neighbouts info indPartInCell and partInCell arrays for SPH and
+      with InCell, IndX and IndY arrays location on the DEM grid
     """
-    # get grid information
+    # get DEM grid information
     header = dem['header']
-    cdef int ncols = header.ncols
-    cdef int nrows = header.nrows
-    cdef float csz = header.cellsize
+    cdef int ncolsDEM = header.ncols
+    cdef int nrowsDEM = header.nrows
+    cdef float cszDEM = header.cellsize
+    # get SPH grid information
+    headerSPH = dem['headerSPH']
+    cdef int ncolsSPH = headerSPH.ncols
+    cdef int nrowsSPH = headerSPH.nrows
+    cdef float cszSPH = headerSPH.cellsize
```

Comment:
Not too important, but if you change code:
ncols -> nCols
nrows -> nRows
cellsize -> cellSize
ncolsDEM -> nColsDEM
etc

Reply:
It is hard to change the fields in the header class, it means a lot of replacement no?
I can do it for the variable we use then